packages/light-api/example/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "example",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@polkadot/light-api": "file:../../",
```

Comment:
was there an issue with yarn workspaces or is this intentional?

Reply:
This is intentional. `@polkadot/light-api` is not (and will not be) in the example's node_modules, so we need to explicitly state where it's located. I'm doing the same thing is js-libs's light.js btw.